springfield/firefox/firefox_details.py

```diff
@@ -22,7 +22,7 @@ def _matches_query(self, info, query):
 
 
 class FirefoxDesktop(_ProductDetails):
-    download_base_url_transition = "/download/thanks/"
+    download_base_url_transition = "/thanks/"
```

Comment:
Haha, you almost got me there pushing this to prod before even merging;D ~ but at least I took it for a spin for a while and love this simplification, good call.